Description: Set $ZSH_HIGHLIGHT_VERSION to the package version
  The $(shell ...) invocations are taken from debian/rules of the
  zsh package[1] with permission.
 .
 [1] git://anonscm.debian.org/collab-maint/zsh.git
Author: Daniel Shahaf <danielsh@apache.org>
Forwarded: not-needed
Last-Update: 2015-10-22
---
--- a/Makefile
+++ b/Makefile
@@ -3,6 +3,8 @@
 INSTALL?=install -c
 PREFIX?=/usr/local
 SHARE_DIR=$(DESTDIR)$(PREFIX)/share/$(NAME)
+DPKG_VENDOR=$(shell dpkg-vendor --query vendor | env LC_ALL=C tr A-Z a-z | env LC_ALL=C tr -d -c '[:alnum:]')
+DCH_VERSION=$(shell dpkg-parsechangelog -SVersion)
 
 # Have the default target do nothing.
 all:
@@ -11,6 +13,8 @@
 install:
 	$(INSTALL) -d $(SHARE_DIR)
 	cp -r .version zsh-syntax-highlighting.zsh highlighters $(SHARE_DIR)
+	printf "%s_%s" "$(DCH_VERSION)" "$(DPKG_VENDOR)" \
+		> $(SHARE_DIR)/.version
 	if [ x"true" = x"`git rev-parse --is-inside-work-tree 2>/dev/null`" ]; then \
 		git rev-parse HEAD; \
 	else \
